doc/publishing-api-syntactic-usage.md

```diff
@@ -3,7 +3,7 @@
 This is the primary interface from publishing apps to the publishing pipeline. Applications PUT items as JSON conforming to a schema specified in govuk-content-schemas.
 Content paths are arbitrated internally by the Publishing API, the content is then forwarded to the live and draft content stores, and placed on the message queue for other apps (eg email-alert-service) to consume.
 
-### Optimistic locking (`previous_version`)
+### [Optimistic locking (`previous_version`)](#optimistic-locking)
```

Comment:
I don't think this syntax does what you think it does: it creates a link to an anchor, it doesn't create an anchor itself. I *think* that Github Flavoured Markdown - and maybe even standard Markdown - automatically creates an anchor for each heading, using the slugified version of the text, so this would be `optimistic-locking-previous-version`.